source/common/protobuf/message_validator_impl.cc

```diff
@@ -24,12 +24,22 @@ void WarningValidationVisitorImpl::onUnknownField(absl::string_view description)
   if (!it.second) {
     return;
   }
+
   // It's a new field, log and bump stat.
-  ENVOY_LOG(warn, "Unknown field: {}", description);
-  if (counter_ == nullptr) {
-    ++prestats_count_;
+  ENVOY_LOG(warn, "Unexpected field: {}", description);
```

Comment:
Why did you change `unknown` to `unexpected` here?

Reply:
Looks like copy paste from previous version when `onUnknownField` and `onDeprecatedField` reused same common method.